src/org/rascalmpl/library/lang/rascal/tests/library/Location.rsc

```diff
@@ -0,0 +1,20 @@
+@license{
+  Copyright (c) 2013-2024 CWI
+  All rights reserved. This program and the accompanying materials
+  are made available under the terms of the Eclipse Public License v1.0
+  which accompanies this distribution, and is available at
+  http://www.eclipse.org/legal/epl-v10.html
+}
+module lang::rascal::tests::library::Location
+
+import Location;
+
+test bool isOverlapping1() =  isOverlapping(|unknown:///|(0, 2), |unknown:///|(0, 2));
```

Comment:
how about moving this to this already existing file with location tests?

https://github.com/usethesource/rascal/blob/cbba43c5477383bedbf7adff276f798247af8f5c/src/org/rascalmpl/library/lang/rascal/tests/basic/Locations.rsc#L437-L448

Reply:
Ok, sure, I'll do.

For future reference, though, which tests go in `lang::rascal::tests::basic::*` and which tests go in `lang::rascal::tests::library::*`? I was assuming that tests related to language primitives would go in `basic::*` and tests for library functions in `library::*`, but I now also notice for instance:

https://github.com/usethesource/rascal/blob/6fa3b438624a199a35a15e76ce09a230881ed093/src/org/rascalmpl/library/lang/rascal/tests/basic/Booleans.rsc#L40

And:

https://github.com/usethesource/rascal/blob/6fa3b438624a199a35a15e76ce09a230881ed093/src/org/rascalmpl/library/lang/rascal/tests/library/Boolean.rsc#L25-L28

Reply:
I have no good answer for this. Maybe @jurgenvinju or @PaulKlint remembers the reason for the split?

In general, I just look at where other functions from the same module are tested.

Reply:
I am the first to admit that this organization can (and should) be improved, but the ratio is this:
- tests for languages features go to `basic`, `functionality` or one of the other more specialized directories
- tests for the standard libraries go to `library`

As a consequence, tests for, say,  locations can be found in two places. Not very good. The problem is that there are several dimensions that can be used to organize the tests:
- syntactic category (e.g., expression, function, pattern, extend, import)
- syntactic alternative (e.g., call, addition, is-defined)
- language feature (e.g., dynamic dispatch, overloading, memoization, parsing)
- data types (e.g., bool, int, map).
- library modules (e.g., Location)

In many cases there is overlap between the categories. **Come up with a better organization and  I will be happy to implement it.**

Reply:
Ok, thanks for clarifying!